ci/jenkins/playbook/jenkins_id_rsa

```diff
@@ -0,0 +1,38 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Not a good idea to checkin private/public keys, better to have it part of github variables

Reply:
will generate once it is running on the runner